suites/remote.js

```diff
@@ -7,6 +7,8 @@ let benchmark = new Benchmarkify("Microservices benchmark").printHeader();
 
 const bench = benchmark.createSuite("Call remote actions");
 
+const natsServerUrl = "nats://" + (process.env.NATS_URL || "localhost:4222")
```

Comment:
Just to note: `NATS_URL` is used there, as `seneca-nats-transport` (btw, deprecated library) is using it anyway, to change NATS server URL.